crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -141,3 +141,26 @@ def foo():
     no_comments
     ):
         pass
+
+match foo:
+    case "a", "b":
```

Comment:
Should this be formatted like this?

```
    case "NOT_YET_IMPLEMENTED_PatternMatchValue", "NOT_YET_IMPLEMENTED_PatternMatchValue":
```

Reply:
not sure tbh. black does not add parentheses even if the line is too long, but i also think it would be good if we would break instead of writing an overlong lines. Also black's behavior is inconsistent with other overlong tuples, e.g. 
```python
match x:
    case "NOT_YET_IMPLEMENTED_PatternMatchValue", "NOT_YET_IMPLEMENTED_PatternMatchValue", "aslödjhkfö":
        pass

b = "NOT_YET_IMPLEMENTED_PatternMatchValue", "NOT_YET_IMPLEMENTED_PatternMatchValue", "aslödjhkfö"
```
becomes
```python
match x:
    case "NOT_YET_IMPLEMENTED_PatternMatchValue", "NOT_YET_IMPLEMENTED_PatternMatchValue", "aslödjhkfö":
        pass

b = (
    "NOT_YET_IMPLEMENTED_PatternMatchValue",
    "NOT_YET_IMPLEMENTED_PatternMatchValue",
    "aslödjhkfö",
)
```

CC @MichaReiser what do you think?

Reply:
I'm not well at the moment and this is taking more brain capacity than I have right now. I hope to get back to this tomorrow

Reply:
> i also think it would be good if we would break instead of writing an overlong lines.

+1

Reply:
The sequence pattern is a bit strange IMO because matching `[a, b]` or `(a, b)` or `a, b` are semantically identical. Meaning, the sequence is neither a list nor a tuple... it's its own thing. 

But I agree, I would expect the sequences to break the same as tuples (and lists) do